Mailer.php

```diff
@@ -0,0 +1,64 @@
+<?php
+// Load Composer's autoloader.
+require __DIR__ . '/vendor/autoload.php';
+require_once __DIR__ . '/MailFunctions.php';
+
+
+// Import PHPMailer classes into the global namespace.
+// These must be at the top of your script, not inside a function.
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\SMTP;
+use PHPMailer\PHPMailer\Exception;
+
+/**
+ * A class to Send mail to a user for otp verification.
+ */
+class Mailer {
+  private $email;
```

Comment:
Can we add some comments to provide information regarding the variables like this

```
  /**
   * @var string $message
   *   Stores User's email id
   */
  private $email;
```